packages/form/src/form.vue

```diff
@@ -85,7 +85,10 @@
       },
       clearValidate(props = []) {
         const fields = props.length
```

Comment:
The type of `props` is expected to be an array. When `props` is a string, `props.length` maybe truthy. I think there's no need to support string param.

Reply:
Accepting a string won't hurt.

Given the source code, from user's perspective: 

* if props is not given or an empty array: clear all fields
* if props is given, and is a non-empty array, clear fields in the array

---

In my opinion, the API of this function should be fixed:

* if props is not given (undefined / null): clear all fields
* if props is a string: clear the field indicated by props
* if props is an array: clear fields indicated by strings in the array

Clear all fields when given an empty array does not make sense to me.